src/vdom.rs

```diff
@@ -153,18 +170,42 @@ impl<Ms: 'static, Mdl: 'static, ElC: View<Ms>, GMs> ::std::fmt::Debug for App<Ms
     }
 }
 
+type InitAppBuilder<Ms, Mdl, ElC, GMs> = AppBuilder<
+    Ms,
+    Mdl,
+    ElC,
+    GMs,
+    (),
+    (),
+    (),
+    InitFn<Ms, Mdl, ElC, GMs>,
+    builder::MountPointInitInitAPI<(), InitFn<Ms, Mdl, ElC, GMs>>,
```

Comment:
Can we reorder types and set default values for some of them to `()`?
The goal is to have something like:
```rust
type InitAppBuilder<Ms, Mdl, ElC, GMs> = AppBuilder<
    Ms,
    Mdl,
    ElC,
    GMs,
    InitFn<Ms, Mdl, ElC, GMs>,
    builder::MountPointInitInitAPI<(), InitFn<Ms, Mdl, ElC, GMs>>,
```

Reply:
Managed to reduce the number of types parameters in another way.